query-grammar/src/user_input_ast.rs

```diff
@@ -205,6 +225,16 @@ impl UserInputAst {
     pub fn or(asts: Vec<UserInputAst>) -> UserInputAst {
         UserInputAst::compose(Occur::Should, asts)
     }
+
+    pub fn set_default_field(&mut self, field: String) {
```

Comment:
Do you really want to commit to this as public API while `UserInputLeaf` is only `pub(crate)`?

Reply:
hum, no I don't